prow/plugins/config.go

```diff
@@ -2030,6 +2067,28 @@ func (l *Label) mergeFrom(other *Label) error {
 	return utilerrors.NewAggregate(errs)
 }
 
+func (b *Blunderbuss) mergeFrom(other *Blunderbuss) error {
```

Comment:
Does the merge finish despite the errors?

Reply:
Yes, because its operating on a pointer, so the updates should be happening directly.